src/__test__/pages/experiments/[experimentId]/plots-and-tables/dot-plot/index.test.jsx

```diff
@@ -99,4 +123,26 @@ describe('Dot plot page', () => {
 
     expect(screen.getByRole('list')).toHaveClass('ant-skeleton-paragraph');
   });
+
+  it('Shows error if there are errors loading cell sets', async () => {
+    const cellSetsErrorResponse = _.merge(
+      {},
+      defaultMockResponses,
+      {
+        [`experiments/${experimentId}/cellSets`]: () => statusResponse(404, 'Nothing found'),
+      },
+    );
+
+    fetchMock.mockIf(/.*/, mockAPI(cellSetsErrorResponse));
+
+    await act(async () => {
+      render(
+        <Provider store={storeState}>
+          {dotPlotPageFactory()}
+        </Provider>,
+      );
+    });
+
+    expect(screen.getByText(/Error loading cell sets/i)).toBeInTheDocument();
```

Comment:
for this test file have you tried what we discussed about checking that tests are doing something? e.g. pass a correct config response in the last one and expect it to get an error (so making sure it fails), and so on

Reply:
yes, I've made sure the tests fail if it's passed the correct response